Testing/Exec/TranEval_FORTRAN/GNUmakefile

```diff
@@ -19,15 +19,15 @@ PELE_PHYSICS_HOME    := ../../..
 USE_F90_PP = TRUE
 
 ifeq ($(FUEGO_GAS), TRUE)
-  Eos_dir         = Fuego
```

Comment:
If the Fortran EOS support is being removed, shouldn't the Eval_FORTRAN test cases also be removed?

Reply:
Good catch @baperry2 . It is done.